install.sh

```diff
@@ -38,6 +38,24 @@ http_get ()
     fi
 }
 
+ln_deps_install ()
+{
+    ln_deps=( \
+        'autoconf' \
+        'automake' \
+        'build-essential' \
+        'git' \
+        'libtool' \
+        'libgmp-dev' \
+        'libsqlite3-dev' \
+        'python3-mako' \
+        'net-tools' \
+        'zlib1g-dev' \
+        'libsodium-dev' \
+        'gettext' )
+    deb_deps_install "${ln_deps[@]}"
```

Comment:
This currently only works for Debian, somebody should look at required Darwin (macOS) deps and add them here too, like in `deps_install()`.

Reply:
Also, `[[ ${use_os_deps_check} != '1' ]]` check should be added too.

Reply:
Partially done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/7a5e4c3369966dda121ac01a09dd999071c8dc4a .

I added the `use_os_deps_check` but for the MacOS i've just kind of "given it a start".

 Probably someone who runs MacOS and can follow the instructions as per the guide [here](https://github.com/ElementsProject/lightning/blob/master/doc/INSTALL.md#to-build-on-macos). From a brief read I note that there are more exceptional conditions to handle here, so it may need a few extra steps.

For now I have just added the basic list of `brew install` dependencies given there, but it's a shot in the dark. @kristapsk feel free to add commits that correct errors in this stuff.